content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -130,10 +130,9 @@ As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/pol
 
 ### Generic ephemeral volumes
 
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
```

Comment:
Hi @pohly . The changes look good.
Can you remove the feature state shortcodes for the alpha and beta states? (lines 133 and 134).
You can list the stable version. Thanks.

Reply:
I've amended the commit, please take another look.

Reply:
@msau42, @kbhawkey: Can I get an new LGTM?